src/www/ui/api/index.php

```diff
@@ -51,16 +53,28 @@
 use Slim\Exception\HttpNotFoundException;
 use Slim\Factory\AppFactory;
 use Slim\Middleware\ContentLengthMiddleware;
+use Slim\Psr7\Request;
 use Slim\Psr7\Response;
 use Throwable;
 
-const REST_VERSION = "1";
+// Extracts the version from the URL
+function getVersionFromUri ($uri)
+{
+  $matches = [];
+  preg_match('/\/repo\/api\/(v\d+)/', $uri, $matches);
```

Comment:
Can just extract the version number here to help making `ApiVersion` enum to use integers instead of strings
```suggestion
  preg_match('/\/repo\/api\/v(\d+)/', $uri, $matches);
```

Reply:
Done @GMishx